C/other/Longest substring without repeating chars.cpp

```diff
@@ -0,0 +1,77 @@
+//This code is submitted to NeoAlgo as a part of Contributors hack 2020

```

Comment:
@Yuvraj-kadale Make all single-line comments in one multiline comment. Something like this
```
/*
...
*/
```

Reply:
Done!!